src/main/java/org/opensearch/knn/plugin/script/KNNScoringSpaceUtil.java

```diff
@@ -21,6 +24,12 @@
 
 public class KNNScoringSpaceUtil {
 
+    private static ModelDao modelDao;
+
+    public static void initialize(ModelDao modelDao) {
```

Comment:
method needs javadoc as it's public

Reply:
all methods that you're adding, at least public ones: getExpectedDimensions, getModelMetadataForField